.travis.yml

```diff
@@ -16,23 +16,31 @@ before_install: git submodule update --init
 
 env:
     global:
+        - CACHE_DIR=build/travis-cache
         - DIST=xenial
         # Make sure beaver is in the PATH
         - PATH="$(git config -f .gitmodules submodule.beaver.path)/bin:$PATH"
 
 install: beaver install
 
+cache:
+    directories:
+        - $CACHE_DIR
```

Comment:
Can you link docs about how does this work? It looks quite different from what I have seen in other CI systems for cache.

Reply:
Yeah, it's abusing the cache because we are not really caching anything, we are just passing artefacts. I could add a link, but I think the same can apply to any construct in this file, so I'm not sure if it objectively makes sense. I could definitely (and probably should too) add a comment to explain a bit more though.

Reply:
I wasn't asking to put it into comment but show it to me because I am not sure how it works :D

Reply:
Updated, also I'm not using more clear variable and directory names.